hedera-node/src/main/java/com/hedera/services/stats/HapiOpCounters.java

```diff
@@ -128,6 +139,10 @@ private void safeIncrement(
 	) {
 		if (!IGNORED_FUNCTIONS.contains(function)) {
 			counters.get(function).getAndIncrement();
+			if (function == ConsensusSubmitMessage) {
+				int txnBytes = txnCtx.accessor().getTxn().getSerializedSize();
+				runningAvgs.recordHandledSubmitMessageSize(txnBytes);
+			}
```

Comment:
Will putting the hook here also update the stat on receiving and submitting the `SubmitMessage` transactions to the network?